setup.py

```diff
@@ -66,22 +73,23 @@ def read(*rnames):
     namespace_packages=['zope'],
     extras_require=dict(
         test=[
-            'zope.app.basicskin',
+            'zope.app.basicskin >= 4.0.0',
             'zope.app.http',
-            'zope.app.pagetemplate',
+            'zope.app.pagetemplate >= 4.0.0',
             'zope.app.principalannotation',
-            'zope.app.publisher',
             'zope.app.publication',
             'zope.app.wsgi',
+
             'zope.applicationcontrol',
             'zope.copypastemove',
             'zope.browser',
             'zope.browsermenu',
             'zope.login',
             'zope.password',
+            'zope.proxy >= 4.2.1',
             'zope.principalregistry',
             'zope.securitypolicy',
-            'zope.testbrowser>5',
+            'zope.testbrowser > 5',
```

Comment:
TBH it's a bit strange to see > rather than >= here, but I suppose it doesn't matter (the current version is 5.2).